docs/modules/ROOT/pages/plugin/how-to-use.adoc

```diff
@@ -17,6 +17,10 @@ Start by adding the dependency `play-soap-plugin` to the project and set require
 `scala` - `scala.concurrent.Future` for Scala projects.
 
 `java` - `java.util.concurrent.CompletionStage` for Java projects.
+|-xjc-Xplay:target
+|play |false |Generate the additional useful classes to using SOAP service with specified framework.
```

Comment:
```suggestion
|play |none |Generates SOAP client classes for the specified framework.
```

Reply:
I changed the text in the last column by your suggestion. But I kept `false` value for `Required` column. WDYT,  @BillyAutrey?

Reply:
LGTM!